README.md

```diff
@@ -1,14 +1,41 @@
-# Knative Sample Controller
+# knative/networking
 
-[![GoDoc](https://godoc.org/knative.dev/sample-controller?status.svg)](https://godoc.org/knative.dev/sample-controller)
-[![Go Report Card](https://goreportcard.com/badge/knative/sample-controller)](https://goreportcard.com/report/knative/sample-controller)
+This repository contains the Knative Ingress and Certificate CRDs, as well as
+their conformance tests. These are our extension points to plugin different
+Ingress plugins (Ambassador, Contour, Gloo, Istio, Kong and Kourier), as well as
```

Comment:
perhaps redirect to docs for the latest list of integrations?

Reply:
Done